codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2Connection.java

```diff
@@ -633,8 +612,7 @@ public void notifyListener(Listener l) {
      * @param events The events (top down order) which have changed
      */
     private void notifyParentChanged(List<ParentChangedEvent> events) {
-        for (int i = 0; i < events.size(); ++i) {
-            ParentChangedEvent event = events.get(i);
+        for (ParentChangedEvent event : events) {
```

Comment:
@Scottmitch +1... We should only use for each if we either use a LinkedList or an array